### ISSUE TITLE
[bug] 1.5.29 fails to install with Yarn

### ISSUE THREAD

**Issue by sqal** [Rel1cx/eslint-react#654]
### Describe the bug

As title says. Yarn can no longer install `@eslint-react/eslint-plugin@1.5.29` with `typescript-eslint` alpha because [YN0001](https://yarnpkg.com/advanced/error-codes#yn0001---exception) error. 

Full log:

```
➤ YN0000: · Yarn 4.3.1
➤ YN0000: ┌ Resolution step
➤ YN0001: │ Error: @typescript-eslint/scope-manager@^7.16.1 || ^rc-v8 isn't supported by any available resolver
    at Pd.getResolverByDescriptor (C:\Users\...\AppData\Local\node\corepack\v1\yarn\4.3.1\yarn.js:141:1698)
    at Pd.bindDescriptor (C:\Users\...\AppData\Local\node\corepack\v1\yarn\4.3.1\yarn.js:141:1083)
    at St.preparePackage (C:\Users\...\AppData\Local\node\corepack\v1\yarn\4.3.1\yarn.js:210:6173)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Z (C:\Users\...\AppData\Local\node\corepack\v1\yarn\4.3.1\yarn.js:210:7587)
    at async Promise.allSettled (index 0)
    at async _c (C:\Users\...\AppData\Local\node\corepack\v1\yarn\4.3.1\yarn.js:140:53129)
➤ YN0000: └ Completed in 0s 709ms
➤ YN0000: · Failed with errors in 0s 729ms
```

package.json

```json
{
  "dependencies": {
    "@eslint-react/eslint-plugin": "1.5.29",
    "typescript-eslint": "^8.0.0-alpha.49"
  },
  "devDependencies": {
    "eslint": "^9.7.0",
    "typescript": "^5.5.3"
  }
}
```

### Expected behavior

Yarn should be able to install the dependencies. 

### Platform and versions

```Text
System:
    OS: Windows 10 10.0.19045
  Binaries:
    Node: 22.4.1 - C:\Program Files\nodejs\node.EXE
    Yarn: 4.3.1 - C:\Program Files\nodejs\yarn.CMD
    npm: 10.8.2 - C:\Program Files\nodejs\npm.CMD
    pnpm: 9.6.0 - C:\Program Files\nodejs\pnpm.CMD
```

**Comment by Rel1cx** [comment]
Fixed this in 15494d75c91e125acd23d0469f796aac48acf623, try installing 1.5.30.

**Comment by sqal** [comment]
All good now 👍 Thanks for quick response.
